contracts/Everlong.sol

```diff
@@ -165,6 +173,22 @@ contract Everlong is IEverlong {
             ? IHyperdrive(_hyperdrive).baseToken()
             : IHyperdrive(_hyperdrive).vaultSharesToken();
 
+        // Ensure target <= 1e18.
+        require(_targetIdleLiquidityPercentage <= 1e18, "Target over 100%");
+
+        // Ensure max <= 1e18.
+        require(_maxIdleLiquidityPercentage <= 1e18, "Max over 100%");
+
+        // Ensure target < max.
+        require(
+            _targetIdleLiquidityPercentage <= _maxIdleLiquidityPercentage,
+            "Target must be <= Max"
+        );
```

Comment:
Let's use custom errors. We don't use string errors in any core contracts. Custom errors have become the industry standard because they are cheaper from a gas and codesize perspective.